content/en/containers/_index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Container Monitoring
```

Comment:
This can be an iterative update, but we should definitely upgrade this landing page to the new style like Serverless and APM have